sympy/physics/control/tests/test_lti.py

```diff
@@ -204,33 +204,30 @@ def test_TransferFunction_addition_and_subtraction():
     tf4 = TransferFunction(p, 2 - p, p)
 
     # addition
-    expect1 = TransferFunction((s - 5)*(s + 3) + (s + 1)*(s + 6), (s - 5)*(s + 1), s)
-    assert tf1 + tf2 == expect1
-    assert tf1 + (s - 1) == TransferFunction(s + (s - 5)*(s - 1) + 6, s - 5, s)
-    assert tf1 + 8 == TransferFunction(9*s - 34, s - 5, s)
-    assert (1 - p**3) + tf1 == TransferFunction(s + (1 - p**3)*(s - 5) + 6, s - 5, s)
+    assert tf1 + tf2 == Parallel(tf1, tf2)
+    assert tf3 + tf1 == Parallel(tf3, tf1)
+    assert -tf1 + tf2 + tf3 == Parallel(-tf1, tf2, tf3)
```

Comment:
How can I make this work based on the way I'm implementing the `Parallel` class? The way it is done in matrices module is that we have a `MatrixExpr` class (see [this](https://github.com/sympy/sympy/blob/c51e328b476402ba16c7e374ac9ddaaa051bc772/sympy/matrices/expressions/matexpr.py#L94)).

Reply:
Same goes for test cases like: `-tf1 + tf2 + tf3 + tf4*tf5 + ... == Parallel(-tf1, tf2, tf3, Series(tf4, tf5), ...)`
Can anyone comment here, please?

Reply:
Naman, I'll have a look later today.

Reply:
`__neg__` will have to handle this, probably by returning `Series(-1, tf1)` for `-tf1`. This is how `MatrixExpr` handles such a thing. (The only difference being matrices evaluates the expression).
I think this type of case would come up for subtractions too.

So it would look like:
```
tf1 + tf2 + tf3 + tf4*tf5 + ... == Parallel(Series(-1, tf1), tf2, tf3, Series(tf4, tf5), ...)
```
But I am not sure about this. It might at times over-complicate the equation.

Reply:
> `__neg__` will have to handle this, probably by returning `Series(-1, tf1)` for `-tf1`.

It was decided to not support integer arguments in `Series`. I think `-tf1*tf2` can be represented as `Series(-tf1, tf2)` & `-tf1 + tf2` can be represented as `Parallel(-tf1, tf2)`, there shouldn't be an issue with that. The problem here is conversion of the transfer function operations like: 

> -tf1 + tf2 + tf3 + tf4*tf5 + ...

into `Parallel` objects.

Reply:
I don't understand what exactly is the problem you are facing. Is it with multiple operators?

Reply:
Yes!

Reply:
For that, you should overload operators (`+, -, *, /`) inside the `Series` and `Parallel` classes too.